src/js/hooks/cycleCount/useIndicatorsTab.jsx

```diff
@@ -31,6 +36,22 @@ const useIndicatorsTab = ({
     facility: currentLocation.id,
   }), [startDate, endDate, currentLocation?.id]);
 
+  useEffect(() => {
+    const currentQueryParams = queryString.parse(history.location.search);
+    const dateParams = startDate && endDate
+      ? {
+        startDate: moment(startDate).format(DateFormat.DD_MMM_YYYY),
+        endDate: moment(endDate).format(DateFormat.DD_MMM_YYYY),
+      }
+      : {};
```

Comment:
can you explain how this was working at all before if we need this change? Why do we need  both the `dateWithoutTimeZone` and the moment format?